resources/compendiums/elemental-skill-particles.md

```diff
@@ -77,32 +82,33 @@ Production ratio shows the rate of the amount of particle that will be produced
+| Klee      | Landing multiple bounce and mines doesn't change the result | 20 | 4 \(-\) | 4.00 | 12.0 |
 | Thoma     |  | 15 | 3~4 \(3:2\) | 3.40 | 13.6 |
-| Yoimiya   | Skill itself produces no particles. Attacks during infusion produce particle \(1 particle, infusion 10s, particle production icd 2s, does not buffer\) | 18 | 4 \(-\) | 4.00 | 13.3 |
+| Xiangling | Off-field \(Guoba\). 1 particle per breathe if enemy survives, particle/min assumes full connect without kills | 12 | 4 \(-\) | 4.00 | 20.0 |
 | Xinyan    | All shield levels produce same particles. Level 3: Rave's Pyro pulse produces no particles | 18 | 4 \(-\) | 4.00 | 13.3 |
-| Klee      | Landing multiple bounce and mines doesn't change the result | 20 | 4 \(-\) | 4.00 | 12.0 |
+| Yanfei    |  | 9 | 3 \(-\) | 3.00 | 20.0 |
+| Yoimiya   | Skill itself produces no particles. Attacks during infusion produce particle \(1 particle, infusion 10s, particle production icd 2s, does not buffer\) | 18 | 4 \(-\) | 4.00 | 13.3 |
 
 {% endtab %}
 {% tab title="Electro" %}  
 
 | Character | Notes | Cooldown \(s\) | Particles \(ratio\) | Average Particles | Particles/min |
 | --------- | :--- | :--- | :--- | :--- | :--- | 
-| Raiden    | Skill itself produces no particles, every type of damage \(including reaction triggered by skill\) triggers coordinated attack, which produce particle \(0~1 \(1:1\) check, 0.9s icd, does not buffer\), doesn't have to stay on field | 0.9 | 0~1 \(1:1\) | 0.50 | 35.0 |
+| Beidou    | **Tidecaller** produce 2 particle at baseline, increase particle by 1 every time a hit is received while holding, max 2. Particle/min assumes always full charge | 7.5 | 2~4 \(-\) | 4.00 | 32.0 | 
 | Fischl    | C6 \(**Evernight Raven**\) Oz, Oz duration doubled with well timed burst | 25 | 0~11 \(-\) | 7.20 | 35.0 |
 | Fischl    | Off-field \(Oz\) \(Oz duration doubled with well timed burst\) | 25 | 0~9 \(-\) | 6.00 | 28.8 |
 | Fischl    | C6 \(**Evernight Raven**\) Oz | 25 | 0~11 \(-\) | 7.20 | 17.3 |
 | Fischl    | Doesn't have to stay on field \(Oz\). Does a 0~1 \(2:3\) check on every Oz's non-constellation 6 non-talent attack, 1s icd interval | 25 | 0~9 \(-\) | 6.00 | 14.4 |
-| Beidou    | **Tidecaller** produce 2 particle at baseline, increase particle by 1 every time a hit is received while holding, max 2. Particle/min assumes always full charge | 7.5 | 2~4 \(-\) | 4.00 | 32.0 | 
-| Razor     |  | 6 | 3 \(-\) | 3.00 | 30.0 |
-| Razor     | Skill Hold  | 10 | 4 \(-\) | 4.00 | 24.0 |
 | Keqing    | Produce particle on leap connect or Thunderclap Slash connect. Both produce same amount of particle | 7.5 | 2~3 \(1:1\) | 2.50 | 20.0 |
 | Lisa      | Skill Hold | 16 | 5 \(-\) | 5.00 | 18.8 | 
+| Raiden    | Skill itself produces no particles, every type of damage \(including reaction triggered by skill\) triggers coordinated attack, which produce particle \(0~1 \(1:1\) check, 0.9s icd, does not buffer\), doesn't have to stay on field | 0.9 | 0~1 \(1:1\) | 0.50 | 35.0 |
```

Comment:
Not true for EC.  Also, this should be $60 / 0.9 = 33.3$ for average particles for minute.  Arguably, you can punish it even moreso via this finding by TWICE https://library.keqingmains.com/evidence/characters/electro/raiden#raiden-coordinated-attack-cooldown (though, it does make the thing overall messy so bleh).